Makefile

```diff
@@ -52,7 +52,7 @@ psql-db-populate:
 psql-db-build-populate: psql-db-build psql-db-populate
 
 .PHONY: psql-db-refresh
-psql-db-refresh: clean-psql psql-db-build-populate
+psql-db-refresh: psql-db-build-populate
```

Comment:
should we keep "clean-psql" as a part of psql-db-refresh? We want to remove any pre-existing database before creating a new one.